### PR TITLE
Update hikamoru.json

### PR DESCRIPTION
**Pull request by ibeswipin**
github pages

## Requirements
Unless explicitly specified otherwise by a **maintainer** or in the requirement description, your domain must pass **ALL** the indicated requirements above.

Please note that we reserve the rights not to accept any domain at our own discretion.

- [✅] The file is in the `domains` folder and is in the JSON format.
- [✅] You have completed your website. 
- [✅] The website is reachable.  
- [✅] You're not using Vercel or Netlify.  
- [✅] The CNAME record doesn't contain `https://` or `/`.  
- [✅] There is sufficient information at the `owner` field.  

## Website Link/Preview

https://amoreforever.github.io/hikamoru/

### CODE REVIEW

File: domains/hikamoru.json [hunk 1]
```diff
@@ -4,6 +4,6 @@
         "email": "me.thefarkhodov@gmail.com"
     },
     "record": {
-        "CNAME": "hikamoru.uz"
+        "CNAME": "amoreforever.github.io/hikamoru/"
```
Comment (21Z) [inline domains/hikamoru.json:7]: `CNAME` can't contain `/`

**Review by 21Z — CHANGES_REQUESTED** [review]

File: domains/hikamoru.json (hunk 1 above)
Comment (AmoreForever) [reply domains/hikamoru.json:7]: why hikamoru.uz isn't working? even hikamoru.uz site is available

Comment (21Z) [reply domains/hikamoru.json:7]: > why hikamoru.uz isn't working? even hikamoru.uz site is available

are you sure you verified the domain on the hosting provider of hikamoru.uz?

Comment (Stef-00012) [reply domains/hikamoru.json:7]: > why hikamoru.uz isn't working? even hikamoru.uz site is available

the current domain shows `ERR_SSL_VERSION_OR_CIPHER_MISMATCH` so it's probably an issue with the SSL

Comment (Stef-00012) [inline domains/hikamoru.json:7]: as 21Z said, CNAME can not contain `/`

**Review by Stef-DP — CHANGES_REQUESTED** [review]

**Pull request closed without merging**